### ISSUE TITLE
[Bug] Trendz Upgrade 1.90-SNAPSHOT to 1.92

### ISSUE THREAD

**Issue by HaydenBurrows** [thingsboard/thingsboard#7732]
Describe the bug
Errors when upgrading from Trendz 1.9.0-SNAPSHOT to Trendz 1.9.2.
Followed procedure on website. https://thingsboard.io/docs/trendz/install/docker/#upgrade-trendz-service

Your Server Environment

    own setup
        docker deployment with AWS RDS Postgres database
        ThingsBoard Version 3.4.1PE
        Ubuntu 22.04

Your Client Environment

    OS: Ubuntu 22.10
    Browser Firefox
    Version 107

To Reproduce

    1.Upgrade from 1.9.0 to 1.9.2 as per website instructions
    2. Observe startup errors
    3. Trendz does not start

Expected behavior
No errors.

Screenshots

**Screenshots**
Upgrade
![image](https://user-images.githubusercontent.com/69659611/205517301-1633f170-88ab-4177-8eb1-2beb644d2762.png)
Errors
![image](https://user-images.githubusercontent.com/69659611/205517120-2340497f-b8b8-4953-90c7-14b20bdcba06.png)

**Comment by vparomskiy** [comment]
Could you share full error logs. I can see that upgrade was performed successfully but I think that Trend app can not start because of issues in trendz.yml (probably merge problem during package upgrade)

**Comment by HaydenBurrows** [comment]
Sure no problem I will get the logs. Would it help if I extract the trendz.yml file from the container?

**Comment by Deesthortered** [comment]
Hello @HaydenBurrows, yes, it probably will be useful. Please, provide the full log file and yml file.

**Comment by HaydenBurrows** [comment]
@vparomskiy apologies for the delay.  I am just back in the office today.  I was thinking about what you mentioned about trendz.yml.  

I checked my docker-compose file and I realised I had a volume which pointed to an external config folder.
This was here from when I was having issues with 0.9.0-SNAPSHOT and Thingsboard 3.4.1 (had to set TB_API_VERSION_MANUAL: 3.3.3x).
![image](https://user-images.githubusercontent.com/69659611/207691367-e03a3f40-f2b9-4949-b8c0-ec6911c9b806.png)


I removed reference to this volume and all is well :)
